hdr_html_canvas_element.md

```diff
@@ -112,77 +114,100 @@ Update `PredefinedColorSpace` to include the following new color spaces.
   }
 ```
 
+#### extended-linear-srgb
+
+The component signals are mapped to red, green and blue tristimulus values according to the following:
+
+* Red primary: `(0.640, 0.330)`
+* Green chromaticity: `(0.300, 0.600)`
+* Blue chromaticity: `(0.150, 0.060)`
+* White chromaticity: `(0.3127, 0.3290)`
+* Transfer function:
+
+```
+   E = | E' / 12.92, if abs(E') ≤ 0.04045
+       | ((E' + 0.055) / 1.055)^2.4, otherwise
+
+       with E' ∈ ℝ
+```
+
+#### extended-srgb
+
+The component signals are mapped to red, green and blue tristimulus values according to the following:
+
+* Red primary: `(0.640, 0.330)`
+* Green chromaticity: `(0.300, 0.600)`
+* Blue chromaticity: `(0.150, 0.060)`
+* White chromaticity: `(0.3127, 0.3290)`
+* Transfer function: `E = E', with E' ∈ ℝ`
+
+#### rec2100-hlg
+
+The component signals are mapped to red, green and blue tristimulus values according to the Hybrid Log-Gamma (HLG) system specified in Rec. ITU-R BT.2100.
+
+#### rec2100-pq
+
+The component signals are mapped to red, green and blue tristimulus values according to the PQ system system specified in Rec. ITU-R BT.2100.
+
 ### Conversion between color spaces
 
-All `PredefinedColorSpace` are defined by how they are converted to XYZD50 under relative colorimetric intent.
+#### General
 
-Converting from XYZD50 to a `PredefinedColorSpace` is done by performing the inverse of the conversion to XYZD50.
+This section specifies the color space conversion when compositing element with color space A onto a canvas element with color space B.
```

Comment:
This sentence needs some work. While the intent is clear, it doesn't use the language of the HTML spec.
* The operation being discussed is [drawing](https://html.spec.whatwg.org/multipage/canvas.html#canvasdrawimage) a [CanvasImageSource](https://html.spec.whatwg.org/multipage/canvas.html#canvasimagesource) in a canvas rendering context.
  * The image source has some color space, and the canvas rendering context has a color space.
  * If they are different, then the image source [must be converted](https://html.spec.whatwg.org/multipage/canvas.html#colour-spaces-and-colour-correction) to the canvas rendering context's before drawing.
* The term "compositing" refers to how things are displayed on the output device (monitor)
  * The general rule is that drawing an image source to a canvas and then compositing that canvas should be directly compositing the image source.
  * This rule cannot be obeyed for HDR content (because it depends on the final display, which is not known for offscreen canvases, and is not knowable due to fingerprinting)
* So what we're discussing is "what is that conversion that is done"
  * For the CSS predefined color spaces (that we want to glom on to here), the HTML specification specifies the conversion [here](https://www.w3.org/TR/css-color-4/#predefined-to-lab) 
  * That definition goes through XYZD50 (and then Lab)

If you want an off the shelf replacement for the above sentence, I'd go with this:

-- 

There are several places in the HTML specification where a color space conversion is required (e.g, when [drawing images to a canvas](https://html.spec.whatwg.org/multipage/canvas.html#colour-spaces-and-colour-correction), [retrieving image data from a canvas](https://html.spec.whatwg.org/multipage/canvas.html#dom-context-2d-getimagedata), among others being added). There exists a standard conversion that is applied to all data in these situations, namely, a conversion using relative colorimetric intent.

This operation is defined for [CSS Predefined Color Spaces](https://www.w3.org/TR/css-color-4/#predefined) in the HTML specification [here](https://www.w3.org/TR/css-color-4/#predefined-to-lab). For each color space, the conversion is parameterized by a transfer function that converts pixel values to a linear space, and a matrix that converts values in that linear space to values in CIE XYZ.

In this section we define this conversion for the new predefined color spaces.